model/canvas.cc

```diff
@@ -629,7 +629,6 @@ namespace minsky
   void Canvas::copyVars(const std::vector<VariablePtr>& v)
   {
     auto group=model->addGroup(new Group);
-    setItemFocus(group);
```

Comment:
I suspect this code can be refactored to not create a new group. Unless we desire to share code between this and paste.